test/lifecycle_test/component2.dart

```diff
@@ -151,10 +151,11 @@ ReactDartComponentFactoryProxy2 PropTypesTest = react.registerComponent(() => ne
 
 class PropTypesTestComponent extends react.Component2 {
   get propTypes => {
-        'intProp': (Map props, propName, componentName, location, propFullName) {
-          if (props[propName] is! int) {
+        'intProp': (Map props, info) {
+          dynamic propValue = props[info.propName];
```

Comment:
#nit prefer using var for local variables
```suggestion
          var propValue = props[info.propName];
```